package org.comstudy21.except;

public class Ch06Ex04 {

	public static void main(String[] args) {
		
		try {
			throw new Exception("사용자 정의 예외 발생!");
		} catch (Exception e) {
			System.out.println(e.getMessage());
		}
		
	}
	
}
